### ISSUE TITLE
Pokecode Awesome Fix einbauen und auf Decidim 0.27.4 upgraden

### ISSUE THREAD

**Issue by bsantschi** [decidim-swiss/decidim.swiss#96]

**Comment by bsantschi** [comment]
Sobald wir von Pokecode das Update des Awesome Moduls (wegen des Fehlers "Admin Verantwortlichkeiten") haben, bitte ich aus Sicherheitsgründen auch gleich noch auf Decidim v0.27.4 upzugraden.

Edit nvb: Awesome Modul statt Schadenmelder. (Hat nichts miteinander zu tun).

**Comment by bsantschi** [comment]
Nadja, lass es uns bitte wissen sobald das Modul ready ist.

**Comment by Naadjjaa** [comment]
@bsantschi Der Fix sollte nun verfügbar sein. Published as Decidim Awesome version 0.9.3.

**Comment by Naadjjaa** [comment]
@carlobeltrame Ich sehe die aktuellsten Funktionen von Awesome nun nicht mehr auf INT (Admin Verantwortlichkeiten und Einstellungen zu der Anzahl Mindestzeichen pro Feld.

**Comment by larsUE** [comment]
@Naadjjaa bei mir ist das alles da (auf den INTs von gemeinsamer.space und Cokreation Winterthur).

**Comment by Naadjjaa** [comment]
@larsUE @carlobeltrame Stimmt, entschuldigt ich habe am falschen Ort gesucht (in Awesome statt Benutzer)

**Comment by Naadjjaa** [comment]
@bsantschi  Habe soeben auf PROD das Admin-Responsability Feature aus dem Awesome Modul ausprobiert und gesehen, dass wir noch immer die Admins der anderen Tenants sehen. Kann es sein, dass ihr auf PROD noch nicht die neuste Version mit dem Fix eingebaut habt?
Edit: Auf INT ebenfalls wieder / immer noch.

**Comment by carlobeltrame** [comment]
@Naadjjaa Auf INT sowie PROD ist Decidim Awesome in der [Version vom 1. August](https://github.com/decidim-ice/decidim-module-decidim_awesome/commits/8c1d23f1ad6433fe6a03bdafb8b5315d9871517c) drauf, 5 Tage nachdem der anscheinend fehlerhafte Admin-Fix dort eingeführt wurde. Ich würde vorschlagen dass du nochmals mit Pokecode / decidim-ice Kontakt aufnimmst um das zu melden.

**Comment by nicolefreypuzzle** [comment]
Bitte Prüfen, ob noch aktuell und in die passende Spalte schieben, etc.
